tests/integration/xml/mcq_with_comments_1.xml

```diff
@@ -12,8 +12,6 @@
             <tip require="elegance" width ="600" height = "800">This is something everyone has to like about this MRQ</tip>
             <tip require="beauty" width ="400" height = "600">This is something everyone has to like about beauty</tip>
             <tip reject="bugs" width = "100" height = "200">Nah, there isn\'t any!</tip>
-
-            <!--<message type="on-submit">Thank you for answering!</message> -->
```

Comment:
Oh, whoops! I didn't notice the name of that file. Will definitely fix :) If I recall, it will just require some weird code due to the conventions of python's XML library when handling comments.